Makefile

```diff
@@ -0,0 +1,53 @@
+
+ifeq ($(CUDA),1)
```

Comment:
So to force a non-CUDA build on a CUDA machine, we would have to set the `CUDA` env var to something different than `1`, right? We should document it somewhere.

Reply:
We can offer both CUDA=1 and CPU=1 switches.
I'm planning to add some documentation via the ARM branch.

Reply:
Ok